doc/developers_guide/developers_guide.md

```diff
@@ -1,3 +1,61 @@
 # Developers Guide
 
 This guide contains information for developers.
+
+## Publishing the Connector to AWS Marketplace
+
+Once the release pushes new connector image to the AWS ECR Registry, you'll have to manually publish the new version.
+
+- Go to Exasol [AWS Marketplace Portal](https://aws.amazon.com/marketplace/management/homepage) (this is main `exasol` account)
+- Go to Products &rarr; Server &rarr; "Exasol Connector for AWS Glue"
+- Click on "Request changes" and select "Add new version"
+- Add new version and for the "Release notes" copy the content of changes markdown file for that version
+- Click on "Activate in AWS Glue Studio" on the Delivery Options section
+- On the "Container image" field, update the **version** at the end of the image
+- On the "Usage instructions" field, update the **version** on the `connectorUrl` and `connectorVersion` properties
+- Only select `Amazon Elastic Container Service (ECS)` on "Supported Services"
+- Click Next
+
+On the next "Restrict version" section, select "No" so that previous version still remain visible to users, and click next.
+
+Finally, review all the changes and submit the request.
+
+## Creating Custom Connector
+
+AWS Glue Studio allows to create a custom connectors. It can be useful to debug the connector.
+
+To test connector by creating a custom connector, please follow these steps.
+
+### Creating an Assembly Jar
+
+By running `mvn verify` or `mvn package` create an connector artifact. For example, `target/exasol-glue-connector-1.0.0-assembly.jar`.
+
+### Uploading the Artifact to S3 Bucket
+
+Upload the JAR artifact from previous step into an S3 bucket. For instance, `s3://exasol-artifacts/glue-connector/`.
```

Comment:
```suggestion
Upload the JAR artifact from previous step into an S3 bucket. For instance, `s3://exasol-artifacts/glue-connector/exasol-glue-connector-1.0.0-assembly.jar`.
```

Reply:
It does not work at the moment. artifact-reference-checker assumes whole text as artifact.
I opened this issue for now: https://github.com/exasol/artifact-reference-checker-maven-plugin/issues/24